### PR TITLE
Helion now as t1 troop transport, unarmed.

### PR DESCRIPTION
**Pull request by trempler**

### CODE REVIEW

File: rules/cabal-vehicles.yaml [hunk 1]
```diff
@@ -393,78 +393,47 @@ TARANTULA:
 SILVER:
 	Inherits@1: ^VoxelVehicle
 	Inherits@2: ^HoverVehicle
-	Inherits@FeatureTargettype: ^FeatureTargettypeAircraft
```
Comment (jrb0001) [inline rules/cabal-vehicles.yaml:396]: No TargetType means that the unit can't be attacked. Is this really what you want?

File: rules/cabal-vehicles.yaml [hunk 2]
```diff
@@ -393,78 +393,47 @@ TARANTULA:
...
-	-Mobile:
-	-LeavesTrails@ground:
-	-LeavesTrails@water:
-	-GrantConditionOnTerrain@Veins:
-	-WithIdleOverlay@Veins:
+	Mobile:
+		TurnSpeed: 5
+		Speed: 80
+	LeavesTrails@ground:
+	LeavesTrails@water:
```
Comment (jrb0001) [inline rules/cabal-vehicles.yaml]: One LeavesTrails should be enough?

File: rules/cabal-vehicles.yaml [hunk 3]
```diff
@@ -393,78 +393,47 @@ TARANTULA:
...
+	GrantConditionOnTerrain@Veins:
```
Comment (jrb0001) [inline rules/cabal-vehicles.yaml]: Is that trait already inherited? If yes, it is useless. If no, it will crash the game.

**Review by jrb0001 — CHANGES_REQUESTED** [review]

File: rules/cabal-vehicles.yaml (hunk 2 above)
Comment (trempler) [reply rules/cabal-vehicles.yaml]: But it is a Hover unit, so there should be Trails on water too? 
Or is @ground for every movement type ?

File: rules/cabal-vehicles.yaml (hunk 3 above)
Comment (trempler) [reply rules/cabal-vehicles.yaml]: Mistake, will fix it.

Comment (jrb0001) [reply rules/cabal-vehicles.yaml]: fixed.

File: rules/cabal-vehicles.yaml (hunk 2 above)
Comment (jrb0001) [reply rules/cabal-vehicles.yaml]: fixed.

File: rules/cabal-vehicles.yaml (hunk 1 above)
Comment (jrb0001) [reply rules/cabal-vehicles.yaml:396]: It is inherited from `^Vehicle`.

**Review by jrb0001 — APPROVED** [review]

**Pull request merged**